src/Cart/Checkout/AddressManagement/AddressesDB.js

```diff
@@ -0,0 +1,22 @@
+export const AddressesDB = [
+  {
+    id: 1,
+    name: "Rishabh",
+    address: "#246, Epitome Colony",
+    city: "Jalandhar",
+    state: "Punjab",
+    country: "India",
+    zip: 152001,
+    mobileNumber: 9877651233
```

Comment:
Mobile number must be of type `string`. The reason is there must be no arithmetic operations happening on this value. The quantity, amount etc. are given type `number` because they are to be undergone arithmetic operations.